Apps/W1/ConnectivityApps/app/src/Codeunits/ConnectivityAppDefinitions.Codeunit.al

```diff
@@ -165,6 +166,34 @@ codeunit 20352 "Connectivity App Definitions"
         AddDescriptionTranslation(AppId, 'Nettbanken din kan kobles til Business Central. Med Continia Payment Management kan du betale dine leverandører, matche kundebetalinger og avstemme kontoutskrifter direkte fra Business Central – fullt integrert og sikkert uten å måtte logge på nettbanken din. Payment Management har integrasjon til følgende banker i Norge: DNB, Handelsbanken, Nordea, SpareBank 1, Sparebanken Vest, Danske Bank. Last ned appen og start din gratis prøveversjon, eller besøk nettsiden vår for mer informasjon.', 1044);
         RegisterApp(AppId, AppName, AppPublisher, AppDescription, AppProviderSupportURL, AppSourceURL, AppApprovedFor, AppWorksOn, "Connectivity Apps Category"::Banking);
     end;
+    
+    local procedure RegisterDirectBankingNL()
+    var
+        AppId: Text[250];
+        AppName: Text[1024];
+        AppPublisher: Text[250];
+        AppDescription: Text[2048];
+        AppProviderSupportURL: Text[250];
+        AppSourceURL: Text[250];
+        AppWorksOn: Text;
+        AppApprovedFor: Text;
+    begin
+        /***************************************************
+            Add app 'Direct Banking' to NL
+        ***************************************************/
+
+        AppId := '7d5b57c9-71d8-47f0-85b8-7a08066f7d2b';
+        AppName := 'Direct Banking NL';
+        AppPublisher := 'IDYN B.V.';
+        AppDescription := 'Integrate ABN, Rabo, ING, Triodos, Knab, ASN, SNS, etc. with Microsoft Dynamics 365 Business Central.';
+        AppProviderSupportURL := 'https://help.idyn.nl/directbanking';
```

Comment:
@RSchoesterIdyn This URL must point to a page where the user immediately sees the list of banks they can connect to through your app. With such page available, the list of supported banks is only one click away from BC. Thanks.

Reply:
Dear @sorenfriisalexandersen: 
I have updated the url for the support page.

However, listing the banks directly on our support page is not possible.
Our payment provider has very frequent updates of their list of supported banks and our support page doesn't allow to embed external pages.
Instead, I added a direct link to the website of our payment provider, which has the full list of supported banks.
So, it's two clicks now.